src/components/UI/navbar/index.js

```diff
@@ -18,24 +27,31 @@ const Navbar = ({ drawerToggleClicked }) => {
         </div>
         <ul>
           <li>
-            <a href="https://www.realdevsquad.com/">Home</a>
+            <a href={HOST_NAME}>
+              <Image
+                width="45px"
+                height="45px"
+                src="/images/Real-Dev-Squad@1x.png"
+                alt="real-dev-squad"
+              />
+            </a>
           </li>
           <li>
-            <a href="https://welcome.realdevsquad.com/">Welcome</a>
+            <a href={WELCOME_LINK}>Welcome</a>
           </li>
           <li>
-            <a href="https://www.realdevsquad.com/events.html">Events</a>
+            <a href={EVENTS_LINK}>Events</a>
           </li>
           <li>
-            <a
-              href="https://members.realdevsquad.com/"
-              style={{ color: '#87D870' }}
-            >
+            <a href={MEMBERS_LINK} style={{ color: '#87D870' }}>
```

Comment:
Did we try giving a className `active` or something instead of CSS in JSX?

Reply:
<a className="active" href={MEMBERS_LINK} >Members</a> .  Like this?

Reply:
i guess it was already in inline css before i changed anything. If you feel that it require a separate class for that css then let me know?